runtime/v1/linux/proc/exec.go

```diff
@@ -142,7 +143,7 @@ func (e *execProcess) Kill(ctx context.Context, sig uint32, _ bool) error {
 
 func (e *execProcess) kill(ctx context.Context, sig uint32, _ bool) error {
 	pid := e.pid.get()
-	if pid != 0 {
+	if pid > 0 {
```

Comment:
I thought the contract of `Kill` on a `zero` pid was `errdefs.ErrNotFound` as the error?

Reply:
resolved